Assets/Scripts/StampProgressManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class StampProgressManager : MonoBehaviour
+{
+    private int numStampsCollected = 0;
+    public TextMeshProUGUI text;
+
+    void OnEnable()
+    {
+        DisplayUpdate();
+    }
+
+    void DisplayUpdate()
+    {
+        numStampsCollected = ProgressManager.Instance.GetNumStampsUnlocked();
+        text.SetText(numStampsCollected.ToString());
+    }
+}

```

Comment:
```suggestion
public class StampProgressManager : MonoBehaviour
{
    public TextMeshProUGUI text;

    void OnEnable()
    {
        DisplayUpdate();
    }

    void DisplayUpdate()
    {
        text.SetText(ProgressManager.Instance.GetNumStampsUnlocked(.ToString());
    }
}

```

Reply:
Just because since we update the variable each time, don't really need to keep a ref to it